examples/getstarted/src/api/address/controllers/address.js

```diff
@@ -0,0 +1,17 @@
+const { createCoreController } = require('@strapi/strapi').factories;
+
+module.exports = createCoreController('api::address.address', ({ strapi }) => ({
```

Comment:
I'm not a huge fan of that syntax. It doesn't look very "clean" for user-land code.

Is this something we could do on our side so that it's invisible for the user/developer? I feel that it might make the code cleaner and less "complicated". Instead of just exporting an object (or a function) you have to remember to call the factory and wrap your object/function in it.

Reply:
You message isn't clear on you you don't like here ? call createCoreController ? passing a function in it ?

Reply:
Sorry, my bad. 

Having to call the `createCoreController`. I feel it adds mental overhead to creating a controller. (same thing for the service). Unless it's an optional thing.

To me it looks like something we should take care for the users. That way they can focus on building their app. It's a small thing that can be generated through the CLI, but it bugs me for some reason.

Reply:
Thanks for the precision, Here is more context

- We do what you suggest for users if they don't have a controller created.
- We also had your suggested behavior until now, but this actually makes the developer experience quite bad. 
- Using a factory actually gives control to users will we keep this easy as we auto-generate it for them anyway.
- This allows exposing IntelliSense features
- This is only necessary when wanting to generate a controller or service with prebuilt CRUD features. If you want to do a 100% custom controller you don't need those utils.
- This makes it possible for plugin developer to use those utils easily